parsers/lib/validation.py

```diff
@@ -129,6 +129,27 @@ def validate_consumption(
     return datapoint
 
 
+def validate_exchange(
+    datapoint: Dict, logger: Union[Logger, None]
+) -> Union[Dict[str, Any], None]:
+    """
+    Validates a production datapoint based on given constraints.
+    If the datapoint is found to be invalid then None is returned.
+    """
+    if logger is None:
```

Comment:
We can put this in the function's signature as you did for format_exchange_data.